scripts/xtensa-build-zephyr.py

```diff
@@ -1190,7 +1217,9 @@ def main():
 
 	build_rimage()
 	build_platforms()
+	create_firmware_bundle()
```

Comment:
```suggestion
	tar_lib_firmware()
```

Reply:
Rename with yet another way to name... it's a tarball.